subprojects/docs/src/samples/userguide/dependencyManagement/troubleshooting/cache/changing/build.gradle

```diff
@@ -2,13 +2,13 @@ apply plugin: 'java-library'
 
 repositories {
     mavenCentral()
-    maven {
-        url 'https://repo.spring.io/snapshot/'
-    }
 }
 
 dependencies {
-    implementation 'org.springframework:spring-web:5.0.3.BUILD-SNAPSHOT'
+    // The snippet is about configuring changing cache, no need to have a real changing module here
+    implementation('org.slf4j:slf4j-api:1.7.25') {
```

Comment:
I'd highly recommend sticking with a real-world snapshot example here. In practice I have rarely seen anyone use `changing = true` and that was only the case for marking dependencies as "changing" that live in Ivy repositories. The far more common use case is that people was use an actual Maven snapshot.

Changing the code here as proposed will have two implications:
1. Users will mark probably mark dependencies with `changing = true` even though they have a `SNAPSHOT` suffix.
2. Users will ask again about how SNAPSHOT versions are handled internally.

I'd personally think it makes far more sense to use a _real_ snapshot version from a _real_ snapshot repository.